src/components/navigation/left-navigation.vue

```diff
@@ -97,7 +128,7 @@ export default {
             q-tooltip(anchor="center right" self="center left" :content-style="{ 'font-size': '1em' }") {{ $t('navigation.left-navigation.proposals') }}
           q-btn.q-ma-md(:class="{'active': activeTab=== LEFT_NAVIGATION_TABS.MEMBERS}" :flat="activeTab !== LEFT_NAVIGATION_TABS.MEMBERS" unelevated rounded padding="12px" icon="fas fa-users" size="sm" :color="activeTab === LEFT_NAVIGATION_TABS.MEMBERS ? 'primary' : 'disabled'" :to="{ name: LEFT_NAVIGATION_TABS.MEMBERS }")
             q-tooltip(anchor="center right" self="center left" :content-style="{ 'font-size': '1em' }") {{ $t('navigation.left-navigation.members') }}
-          q-btn.q-ma-md(:class="{'active': activeTab=== LEFT_NAVIGATION_TABS.ELECTION}" :flat="activeTab !== LEFT_NAVIGATION_TABS.ELECTION" unelevated rounded padding="12px" size="sm" :color="activeTab === LEFT_NAVIGATION_TABS.ELECTION ? 'primary' : 'disabled'" :to="{ name: LEFT_NAVIGATION_TABS.ELECTION }")
+          q-btn.q-ma-md(v-if="elections.length" :class="{'active': activeTab=== LEFT_NAVIGATION_TABS.ELECTION}" :flat="activeTab !== LEFT_NAVIGATION_TABS.ELECTION" unelevated rounded padding="12px" size="sm" :color="activeTab === LEFT_NAVIGATION_TABS.ELECTION ? 'primary' : 'disabled'" :to="{ name: LEFT_NAVIGATION_TABS.ELECTION }")
```

Comment:
> @evgenibir lets try to avoid having direct values in HTML because they create a lot of bugs. I think in this case we can have computed with something like `hasElection`

fixed